### ISSUE TITLE
maven build is failing

### ISSUE THREAD

**Issue by coolsan24** [MangoAutomation/BACnet4J#65]
Failed to read artifact descriptor for com.serotonin:bacnet4j:jar:4.1.6: Could not transfer artifact com.serotonin:bacnet4j:pom:4.1.6 from/to ias-releases (https://maven.mangoautomation.net/repository/ias-release/): PKIX path validation failed: java.security.cert.CertPathValidatorException: validity check failed: NotAfter: Wed Dec 01 23:59:59 UTC 2021 -> [Help 1]

**Comment by jinuxx** [comment]
same here, seems that the maven repo is broken.

**Comment by coolsan24** [comment]
yeah , I think they need to fix it soon ,... anybody can you check this please

**Comment by vmustafayev4Eqnx** [comment]
Hi, seems like the certificate expired. Could anyone fix it ASAP?
Thanks in advance

**Comment by mlohbihler** [comment]
Closing due to lack of activity
